app/bundles/EmailBundle/Tests/EventListener/PointSubscriberFunctionalTest.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\FormBundle\Tests\Controller;
+
+use Mautic\CategoryBundle\Entity\Category;
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\EmailBundle\Entity\Email;
+use Mautic\EmailBundle\Entity\Stat;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Entity\LeadRepository;
+use Mautic\PointBundle\Entity\Point;
+
+class PointSubscriberFunctionalTest extends MauticMysqlTestCase
+{
+    /**
+     * @var LeadRepository
+     */
+    private $contactRepository;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->contactRepository  = $this->em->getRepository(Lead::class);
```

Comment:
Yes, this sucks. I did not find other way around it than something like:
```suggestion
        /** @var LeadRepository $contactRepository;
        $contactRepository = $this->em->getRepository(Lead::class);
        $this->contactRepository  = $contactRepository;
```